apps/ledger-live-desktop/src/renderer/components/ToastOverlay/Toast.jsx

```diff
@@ -117,16 +117,17 @@ export function Toast({
     }
   }, [duration, id, onDismiss]);
 
+  const onClick = useCallback(
+    event => {
+      callback && callback();
```

Comment:
This is why so many people hate javascript 😂 

if (callback) {
   callback();
}

🙏